dnf/cli/main.py

```diff
@@ -87,6 +89,8 @@ def _main(base, args, cli_class, option_parser):
 
     # our core object for the cli
     base._logging._presetup()
+    # TODO: move somewhere I can check configuration options
+    dnssec.RpmImportedKeys.check_imported_keys_validity()
```

Comment:
move to base.py - fill_sack(), probably somewhere under "if load_available_repos:"
we want dnssec checking only if we load remote repositories and work with them
revoking keys makes sense only before downloading and installing something from repos,
having revoked keys on system is ok for remove, local queries, etc.